src/SelectableBox/SelectableBox.scss

```diff
@@ -0,0 +1,56 @@
+@import "variables";
+
+.pgn__selectable_box-set {
+  display: flex;
+  flex-wrap: wrap;
+  flex-direction: row;
+  justify-content: space-between;
+  width: auto;
+  min-height: $selectable-box-set-height;
+  margin-right: -$selectable-box-space;
+
+  & > * + * {
+    margin: 0;
+  }
+
+  & > * {
+    margin-top: $selectable-box-space;
+    margin-right: $selectable-box-space;
+  }
+}
+
+.pgn__selectable_box {
+  min-height: inherit;
+  flex: 1;
+  position: relative;
+  padding: $selectable-box-padding;
+  box-shadow: 0 1px 2px rgba(0, 0, 0, 0.15), 0 1px 4px rgba(0, 0, 0, 0.15);
```

Comment:
It looks like the Figma specifies the shadow as using level-1 shadow; we should use the `$level-1-box-shadow` SCSS variable here. The box shadow depicted in the Figma is for the brand-edx.org theme which has the appropriate `$level-1-box-shadow` theme variable override.

Reply:
right, I didn't see this variable and just copied styles from Figma. Fixed now